tools/gulp/packaging/build-config.ts

```diff
@@ -0,0 +1,17 @@
+/** Interface for the build config. */
+export interface BuildConfig {
+  projectVersion: string;
+  projectDir: string;
+  packagesDir: string;
+  outputDir: string;
+  licenseBanner: string;
+}
+
+/** Build config being used to build package output. */
+export let buildConfig: BuildConfig;
+
+/** Configures the build by setting the build config. */
+export function configureBuild(config: BuildConfig) {
```

Comment:
The reason I wanted to keep the config exposed is that other custom tasks (that can be specific to material or flex-layout) may need access to the build config for creating their paths.